core/tests/protractor_desktop/embedding.js

```diff
@@ -239,7 +239,14 @@ describe('Embedding', function() {
     expect(embeddingLogs).toEqual(expectedLogs);
 
     await users.logout();
-    await general.checkForConsoleErrors([]);
+
+    // This error is to be ignored as 'idToBeReplaced' is not a valid
```

Comment:
How did it work earlier?

Reply:
Earlier it was resulting into this failure (https://github.com/oppia/oppia/runs/2977880922?check_suite_focus=true#step:12:719)

Reply:
I noticed that we seem to be using this technique already:

https://github.com/oppia/oppia/blob/a2f9d1465f8676e9779b0b8c33ba178e374ff525/core/tests/protractor_desktop/embedding.js#L337-L341

so approving. Thanks!

Reply:
But this technique was adopted when it was failing everytime in the past. Although the flake which I was working in this PR is very similar but this time the failure is not happening repeatdely . So, we thought it would be better if we could get the correct reason behind the flake

Reply:
Ah I see. Thanks for clarifying! I'm removing my LGTM for now, in that case. Please lmk once you have identified the root cause of the issue and I will review it again.